### PR TITLE
Feat: setup issue template

### PR DESCRIPTION
**Pull request by samyakshah3008**
## Description: 

- Added a template for someone who is creating a new issue ticket. This will help us to maintain the consistency across the project and we can make sure no information gets missed to be added. 

This PR closes issue  #17

### CODE REVIEW

**Review by ardourApeX — COMMENTED** [review]
You can consider adding these fields also, if it makes sense to you.

### Reproducibility

- [ ] This issue is reproducible
- [ ] This issue is not reproducible

#### Steps to Reproduce

1. [Step 1]
2. [Step 2]

### Severity/Priority

- [ ] Critical
- [ ] High
- [ ] Medium
- [ ] Low

### Labels  
- [ ] Bug 
- [ ] Feature Request 
- [ ]  Enhancement 
- [ ]  Documentation 
- [ ]  Help Wanted

**Comment by samyakshah3008** [comment]
Thanks @ardourApeX for your review, I have added your suggested fields.

**Review by ardourApeX — APPROVED** [review]

File: .github/Issue_Template.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ - [ ] Medium
+ - [ ] Low
+
+### Labels
+ - [ ] Bug
+ - [ ] Feature Request
+ - [ ] Enhancement
+ - [ ] Documentation
+ - [ ] Help Wanted
+ - [ ] Others
```
Comment (vinit717) [inline .github/Issue_Template.md:38]: no need to add a label here because we have an inbuild GitHub label already

**Comment by samyakshah3008** [comment]
@ardourApeX @vinit717 please check again, cheers.

**Review by ardourApeX — APPROVED** [review]

**Review by iamitprakash — APPROVED** [review]

**Comment by samyakshah3008** [comment]
@vinit717 can you please resolve the conversations? I have worked on your requested suggestions.

**Review by vinit717 — APPROVED** [review]

**Pull request merged**